pypesto/visualize/sampling.py

```diff
@@ -92,119 +112,347 @@ def sampling_fval_trace(
     return ax
 
 
-def sampling_prediction_profiles(result: Result,
-                                 alpha: Sequence[int] = None,
-                                 stepsize: int = 1,
-                                 plot_type: str = 'states',
-                                 title: str = None,
-                                 size: Tuple[float, float] = None,
-                                 ax: matplotlib.axes.Axes = None):
+def _get_percentile_cutoffs(percentiles: float):
+    half_difference = (100 - percentiles)/2
+    return {'lower': half_difference, 'upper': 100 - half_difference}
+
+
+def sampling_prediction_trajectories(
+        ensemble_prediction: EnsemblePrediction,
+        percentiles: Union[float, Sequence[float]],
+        title: str = None,
+        size: Tuple[float, float] = None,
+        axes: matplotlib.axes.Axes = None,
+        labels: Dict[str, str] = None,
+        axis_label_padding: int = 30,
+        groupby: str = CONDITION,
+        condition_gap: float = 0.01,
+):
     """Plot MCMC-based prediction confidence intervals for the
     model states or observables. One or various confidence levels
-    can be depicted.
+    can be depicted. Plots are grouped by condition.
+
+    Note that references to observables here corresponds to the meaning of an
+    observable in the pyPESTO predictions code, which means an observable could
+    refer to many things, including model states or observables, dependent on
+    how the prediction is processed. FIXME rename
 
     Parameters
     ----------
     result:
         The pyPESTO result object with filled sample result.
-    alpha:
-        List of lower tail probabilities, defaults to 95% interval.
-    stepsize:
-        Only one in `stepsize` values is simulated for the intervals
-        generation. Recommended for long MCMC chains. Defaults to 1.
-    plot_type:
-        Visualization mode for prediction intervals {‘states’, ‘observables’}.
-        Defaults to ‘states’.
+    percentiles:
+        Lower tail probabilities, e.g. `[95]` for a 95% credibility interval.
```

Comment:
"Confidence levels"? And maybe note that these are translated into symmetric intervals, e.g. 95% -> 2.5% - 97.5%. An alternative would be e.g. highest-density regions.